src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -932,111 +943,18 @@ x3dom.registerNodeType(
                                             this._mesh._texCoords[0].push(texCoords[t2].z);
                                         }
                                     }
-
```

Comment:
I edited the explanation to make it easier.
I agree with you, it is safer to test this before merging it.